README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
I'm fine if you don't want this or want it somewhere else.  Putting it in README does make it show on the github page for any branch with the badge.

People trying to install a version of wagn/decko will want this information.  They won't want to install code when this isn't green.  It should always be green on master, and if it isn't a core dev better fix it soon.